### PR TITLE
InoReader API Init

### PR DESCRIPTION
**Pull request by kanishkablack**
InoReader

### CODE REVIEW

File: schemas/org.gnome.feedreader.inoreader.gschema.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<schemalist>
+	<schema id="org.gnome.feedreader.inoreader" path="/org/gnome/feedreader/inoreader/" gettext-domain="inoreader">
+	<key name="inoreader-api-token" type="s">
+		<default>""</default>
+		<summary>Acces key to Feedly-API</summary>
```
Comment (bil-elmoussaoui) [inline schemas/org.gnome.feedreader.inoreader.gschema.xml]: Summary strings should be changed to InoReader

Comment (kanishkablack) [inline schemas/org.gnome.feedreader.inoreader.gschema.xml]: @bil-elmoussaoui i didn't follow you

Comment (bil-elmoussaoui) [inline schemas/org.gnome.feedreader.inoreader.gschema.xml]: The summary string is still `Acces key to Feedly-API` it should be `Acces key to InoReader API`

**Comment by jangernert** [comment]
Why does the user need to enter an api-key and secret?
I know there are some api call limitations based on the api key.
But having more requirements to log in than simply username and password is bad UX.

**Comment by kanishkablack** [comment]
@jangernert there is an API limit each api key, i haven't contacted inoreader for extending the limit.
i will looking into it once and update the code base

**Comment by jangernert** [comment]
I will close this pull request since I have set up a inoreader branch in the main repository and granted you commit access there.

**Pull request closed without merging**